### PR TITLE
Add a command to install radare2

### PR DESCRIPTION
**Pull request by qbarrand**

### CODE REVIEW

File: pkg/site/r2_windows.go [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"golang.org/x/xerrors"
+)
+
+func (s Site) InstallRadare2(prefix string) error {
+	url := "http://radare.mikelloc.com/get/3.6.0/radare2-msvc_64-3.6.0.zip"
```
Comment (xarkes) [inline pkg/site/r2_windows.go]: Maybe you should put a variable for the version so it's easier to upgrade it?
By the way, http://radare.mikelloc.com/get/3.9.0/radare2-msvc_64-3.9.0.zip is out.

Comment (qbarrand) [reply pkg/site/r2_windows.go]: Absolutely. Ideally, not specifying a version should pull the latest release, but I don't see any obvious way to do that with http://radare.mikelloc.com. I think we have to make the CLI flag mandatory then.

Comment (XVilka) [reply pkg/site/r2_windows.go]: You can take it from AppVeyor directly though, see the API example in the release script: https://github.com/radare/radare2-release/blob/master/build.sh#L515

Comment (qbarrand) [reply pkg/site/r2_windows.go]: Can I actually do that?

```
$ curl -i https://ci.appveyor.com/api/projects/radare/radare2-shvdd
HTTP/2 404
cache-control: no-cache
pragma: no-cache
content-type: application/json; charset=utf-8
expires: -1
x-xss-protection: 1; mode=block
x-frame-options: SAMEORIGIN
strict-transport-security: max-age=31536000
x-content-type-options: nosniff
date: Wed, 18 Sep 2019 20:42:35 GMT
content-length: 49

{"message":"Project not found or access denied."}%
```

From https://github.com/radare/radare2-release/blob/8062c0d4dcfb9919c88cd0b5cb00652bfa09b826/build.sh#L538

Comment (Maijin) [reply pkg/site/r2_windows.go]: yes that is due to the migration from https://github.com/radare/radare2 to https://github.com/radareorg/radare2 @XVilka is working on it

Comment (XVilka) [reply pkg/site/r2_windows.go]: @qbarrand @Maijin the change is trivial, just use https://github.com/radare/radare2-release/pull/29

**Comment by XVilka** [comment]
Is it good to merge? @qbarrand

File: pkg/site/r2_windows.go (hunk 1 above)
Comment (XVilka) [reply pkg/site/r2_windows.go]: Can you please fix this before the merge?

**Comment by XVilka** [comment]
Ping?

**Comment by XVilka** [comment]
Ping?

**Comment by qbarrand** [comment]
> Ping?

Is AppVeyor disabled now? It's still triggered... Thanks

**Comment by xarkes** [comment]
I disabled it for that repository.

**Pull request merged**